transport/controller.cc

```diff
@@ -70,6 +71,14 @@ future<> controller::do_start_server() {
         cql_server_config.timeout_config = make_timeout_config(cfg);
         cql_server_config.max_request_size = service::get_local_storage_service()._service_memory_total;
         cql_server_config.max_concurrent_requests = cfg.max_concurrent_cql_requests();
+        _observers.start().get();
+        _observers.invoke_on_all([this, cserver] (std::vector<std::any>& observers) {
+            auto& cfg = _db.local().get_config();
+            observers.emplace_back(make_lw_shared<utils::observer<uint64_t>>(cfg.max_concurrent_cql_requests.observe([cserver] (size_t value) {
+                logger.info("Setting max concurrent CQL requests threshold to {}", value);
+                cserver->local().set_max_concurrent_requests(value);
+            })));
```

Comment:
You can also use updateable_value. It is designed to be move-constructible os you can just place it in config. It is also cross-shard-safe.

Reply:
done